bpf/bpf_network.c

```diff
@@ -79,10 +87,12 @@ int cil_from_network(struct __ctx_buff *ctx)
 
 out:
 	send_trace_notify(ctx, obs_point_from, 0, 0, 0,
-			  ctx->ingress_ifindex, reason, TRACE_PAYLOAD_LEN);
+			  ingress_ifindex,
+			  trace.reason, trace.monitor);
```

Comment:
I wouldn't do it as part of this fix, but might be nice if the last arg was just the trace_ctx?

Reply:
I agree, that would be nice. Would you be up for creating an issue? It could probably be tagged as a good first issue.